packages/in_app_purchase/in_app_purchase_storekit/lib/src/in_app_purchase_storekit_platform.dart

```diff
@@ -68,10 +68,10 @@ class InAppPurchaseStoreKitPlatform extends InAppPurchasePlatform {
   Future<bool> isAvailable() => SKPaymentQueueWrapper.canMakePayments();
 
   @override
-  Future<bool> buyNonConsumable({required PurchaseParam purchaseParam}) async {
```

Comment:
Don't we need to change the function signature of the parent class too?
And it looks like quantity should be part of PurchaseParam as well.

Reply:
Thanks for your comment. According to your suggestions I will move the quantity param to PurchaseParam and change the function signature of the parent class. Then I will create issue describing the feature.